core/src/main/java/com/klaytn/caver/contract/ContractMethod.java

```diff
@@ -98,6 +98,23 @@ public ContractMethod(Caver caver, String type, String name, List<ContractIOType
         this.contractAddress = contractAddress;
     }
 
+    /**
+     * Execute smart contract method in the EVM without sending any transaction.
+     * When creating CallObject, it need not to fill 'data', 'to' fields.
+     * The 'data', 'to' fields automatically filled in call() method.
+     * @param arguments A List of parameter to call smart contract method.
+     * @return List
+     * @throws IOException
+     * @throws ClassNotFoundException
+     * @throws NoSuchMethodException
+     * @throws InstantiationException
+     * @throws IllegalAccessException
+     * @throws InvocationTargetException
+     */
+    public List<Type> call(List<Object> arguments) throws IOException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException {
```

Comment:
Can you add test code for this? to confirm this will work fine?

Reply:
I added test case.